amd/kvTrack.js

```diff
@@ -21,14 +21,25 @@ function kvTrack(uaID) {
 
 	// Initialize Google Analytics
 	self.setUAId(uaID);
-	self.initGA();
+
+	// Use existing lib if present
+	if (typeof ga === 'function') {
+		this.ga = ga;
```

Comment:
Should this be `self.ga = ga;`?

Reply:
Pretty sure we only need to use the `self` reference when we're within one of the jQuery wrapped methods so most of these could be cleaned up and just use `this`.

I definitely copied over some that needn't reference the copy from within the $.ajax method.